convex/modelParameterHelpers.ts

```
@@ -0,0 +1,291 @@
+  apiParameters: ApiParameters,
+  mappingLog: string[]
+): void {
+  for (const [frontendKey, apiKey] of Object.entries(mappings)) {
+    const frontendValue =
+      frontendParams[frontendKey as keyof FrontendParameters];
+
+    if (frontendValue !== undefined && frontendValue !== null) {
+      // Special handling for different parameter types
+      switch (frontendKey) {
+        case "duration":
+          const durationValue = parseInt(frontendValue as string);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)